android/app/src/main/res/values-vi/strings.xml

```diff
@@ -588,7 +588,6 @@
 	<string name="phone_number">Số điện thoại</string>
 	<string name="profile">Hồ sơ OpenStreetMap</string>
 	<plurals name="bookmarks_detect_message">
-	  <item quantity="one">%d dã tìm thấy một tệp. Bạn sẽ thấy nó sau khi chuyển đổi.</item>
```

Comment:
The source of truth is data/strings/strings.txt, so translations should be removed from there, and then, in a separate commit, these files should be updated using tools/unix/generate_localizations.sh script.

But let's clarify first the difference in translation with someone who knows these languages.

Reply:
There are two commits, I followed the translation docs for that.

Digging into git blame, it would seem these where all landed as one in  e4bdc5526aa06caa7834b39be39b8b8b18b43755